src/main/io/serial.c

```diff
@@ -293,6 +302,21 @@ serialPort_t *openSerialPort(
             serialPort = uartOpen(USART3, callback, baudRate, mode, options);
             break;
 #endif
+#ifdef USE_USART4
+    case SERIAL_PORT_USART4:
+        serialPort = uartOpen(UART4, callback, baudRate, mode, options);
```

Comment:
I was about to do something similar for the F3 boards and noticed something. There is absolutely no consistency with how uart and usart are used (granted nothing we do uses usart).

Going forward, I think it would be much easier if every instance was changed to one or the other, as you see here we have UART4, 5 is 'USART', I know on the F3, *serial5* is actually a UART. You can see how this would get confusing.

Why not use this as an opportunity to remove uart/usart terms completely, go with something generic like SERIAL# possibly.


Then per chip or target, it would just be mapping SERIAL1 to the correct 'UART/USART' from the stdperiph library.